messages/rest.md

```diff
@@ -81,4 +81,8 @@ HTTP header in "key:value" format.
 
 # flags.body.summary
 
-File or content for the body of the HTTP request. Specify "-" to read from standard input or "" for an empty body. If passing a file, prefix the filename with '@'. 
+File or content for the body of the HTTP request. Specify "-" to read from standard input or "" for an empty body. If passing a file, prefix the filename with '@'.
+
+# flags.no-auth.summary
+
+Skip the "Authentication" header.
```

Comment:
```suggestion
Skip adding the "Authorization: Bearer <token>" header to requests.
```

Reply:
Okay?  I copied your text from the PR description verbatim!   But I like that it adds more detail.